### ISSUE TITLE
Issues with download path

### ISSUE THREAD

**Issue by Waxes27** [Daemitrious/9anime_dl#5]
Please indicate download path:

	> ~/Videos

Please indicate download path:

**Comment by Waxes27** [comment]
Seems like the "~" isn't supported, do I have pushing permissions to make adjustments

**Comment by Daemitrious** [comment]
@Waxes27 yeah if you could figure that out that would be awesome because Im honestly not too familiar with using the tilde with linux.
